openstef_dbc/services/prediction_job.py

```diff
@@ -4,38 +4,14 @@
 
 import json
 from typing import List, Optional, Union
-from pydantic import BaseModel, ValidationError
+from pydantic import ValidationError
+from openstef.data_classes.prediction_job import PredictionJobDataClass
```

Comment:
openstef_dbc does not have openstef dependency (in requirements.txt) right? So I don't think this import works. 

Plus now we changed the dependency. First openstef was dependent on openstef_dbc. Now openstef_dbc is dependent on openstef. This is already a step in a good direction, but I would prefer to have no dependency from openstef_dbc on openstef. Then applications (e.g. opwek-inschatter) can use openstef_dbc too without importing openstef. We could achieve this by directly giving a correct prediction_job dictionary from openstef to openstef_dbc methods in `PredictionJobRetriever`. Then `PredictionJobRetriever` in openstef_dbc does not need to check the Pydantic Datamodel `PredictionJobDataClass` from openstef.

Reply:
lets tackle this in a new US